framework/xds_flags.py

```diff
@@ -190,6 +190,12 @@
     help="Whether to enable GFE debug headers and what value to use.",
 )
 
+ADD_DUALSTACK_SUPPORT = flags.DEFINE_bool(
+    "add_dualstack_support",
```

Comment:
Here and elsewhere: but let's name it `enable_dualstack`. 
Sorry, we should've suggested it much sooner, so now there's plenty of renaming todo.

Reply:
Done everywhere